test/Microsoft.TestPlatform.Extensions.TrxLogger.UnitTests/TrxLoggerTests.cs

```diff
@@ -712,7 +712,6 @@ private void ValidateDateTimeInTrx(string trxFileName)
 
         [TestMethod]
         [DataRow("results")]
-        [DataRow("results.trx")]
```

Comment:
instead add separate test, use data row with 2 arguments, one is the input and other expected output

Reply:
'.' is now like other alphabets right? do we really need another test case?